producer/src/main/java/controller/NewCarsSender.java

```diff
@@ -35,13 +36,17 @@ public static void main(String[] args) throws InterruptedException {
         Producer<String, Car> producer = new KafkaProducer<String, Car>(props);
 
         while (true) {
-            //Creating new car
             Car car = new Car();
-            carService.makeCar(car);
+            Socket socket = new Socket("localhost", 4444);
+            ObjectOutputStream os = new ObjectOutputStream(socket.getOutputStream());
```

Comment:
why do you need to send anything to server at all? Client should only receive cars!

Reply:
I deleted it